app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -3198,9 +3204,8 @@ public void selected(final PlayQueueItem item, final View view) {
 
             @Override
             public void held(final PlayQueueItem item, final View view) {
-                final int index = playQueue.indexOf(item);
-                if (index != -1) {
-                    playQueue.remove(index);
+                if (playQueue.indexOf(item) != -1) {
+                    heldQueueItem(item, view);
```

Comment:
Just as you did here, please also change
https://github.com/TeamNewPipe/NewPipe/blob/a2297fb5b8b7908ff3c2522dd5395b76fdfee763/app/src/main/java/org/schabi/newpipe/player/PlayQueueActivity.java#L372-L379
to
```java
if (player != null && player.getPlayQueue().indexOf(item) != -1) {
    openPopupMenu(/* ... */);
}
```